media-gfx/alembic/alembic-1.7.11.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils multiprocessing python-single-r1
+
+DESCRIPTION="Open framework for storing and sharing scene data"
+HOMEPAGE="https://www.alembic.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="arnold +boost doc examples hdf5 maya prman python test zlib"
+
+# pyalembic python bindings need boost
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	hdf5? ( zlib )
+	python? ( boost )
+"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=media-libs/openexr-2.3.0:=
+	boost? ( >=dev-libs/boost-1.65.0:=[python,${PYTHON_USEDEP}] )
+	hdf5? ( >=sci-libs/hdf5-1.10.2:=[zlib(+)] )
+	python? ( >=dev-python/pyilmbase-2.3.0[${PYTHON_USEDEP}] )
+	zlib? ( >=sys-libs/zlib-1.2.11-r1:= )
```

Comment:
that slot op is not necessary, introducing subslot on zlib was a mistake.

Reply:
Zlib does have subslots, so why don't we need a subslot operator in this case?

Reply:
package having subslot does not always automatically mean we need to add a slot operator on all revdeps (see poppler, see Qt packages). in this case, as said, it was simply a mistake introducing it.

Reply:
Ok, thanks for explanation :)